products/dns/terraform.yaml

```diff
@@ -190,6 +190,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       pre_delete: templates/terraform/pre_delete/detach_network.erb
   ResourceRecordSet: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  ResourceDnsRecordSet: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project}}/managedZones/{{managed_zone}}/rrsets/{{name}}/{{type}}"]
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      encoder: 'templates/terraform/encoders/dns_resource_record_set.go.erb'
+      constants: 'templates/terraform/constants/resource_dns_resource_record_set.go.erb'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        min_version: beta
+        name: "dns_resource_dns_record_set_basic"
+        primary_resource_id: "resource-recordset"
+        vars:
+          zone_name: "my-zone"
+    properties:
+      # excluding kind from Terraform code because this is a static, hardcoded value
```

Comment:
We should either exclude `kind` or just remove it from schema right?

Reply:
@venkykuberan what about this point? We automatically set kind in the custom encoder, shouldn't we exclude it?

Reply:
@c2thorn i don't see that entry anymore on my local branch after i pushed the change, not sure why its still showing up here. Any thoughts ?

Reply:
@venkykuberan yeah I reopened a comment on an outdated commit, because it was marked as resolved without addressing what I was talking about.

I believe we should add an override to exclude the `kind` field

Reply:
removed the `kind` attribute reference as it doesn't require in API request